src/tribler-core/tribler_core/components/metadata_store/db/store.py

```diff
@@ -626,7 +626,7 @@ def get_entries_query(
         if cls is None:
             cls = self.ChannelNode
         pony_query = self.search_keyword(txt_filter, lim=1000) if txt_filter else left_join(g for g in cls)
-
+        infohash_set = {infohash} if isinstance(infohash, bytes) else infohash
```

Comment:
I also don't like the idea of overusing the parameter. For me, it looks more correct to replace `infohash` by `infohash_list` but it is hard to determine all usages of the parameter because it normally passes as a `**kwarg`.

I'll follow your suggestion because it is better than overusing.

Reply:
Added separate parameter `infohash_set`